src/families/stellar/api/horizon.js

```diff
@@ -0,0 +1,266 @@
+      .call();
+
+    mergedTransactions = transactions.records;
+
+    while (transactions.records.length > 0) {
+      transactions = await transactions.next();
+      mergedTransactions = mergedTransactions.concat(transactions.records);
+    }
+  } catch (e) {
+    return [];
+  }
+
+  return mergedTransactions;
+};
+
+const fetchOperationList = async (
+  accountId: string,
+  addr: string,
+  transactions: RawTransaction[]
+): Promise<Operation[]> => {
+  let formattedMergedOp = [];
+
+  for (let i = 0; i < transactions.length; i++) {
+    let operations = await server
+      .operations()
+      .forTransaction(transactions[i].id)
+      .limit(LIMIT)
+      .call();
+
+    formattedMergedOp = formattedMergedOp.concat(
+      rawOperationToOperation(
+        operations.records,
+        transactions[i],
+        addr,
+        accountId
+      )
+    );
+
+    while (operations.records.length > 0) {
+      operations = await operations.next();
+
+      formattedMergedOp = formattedMergedOp.concat(
+        rawOperationToOperation(
+          operations.records,
+          transactions[i],
+          addr,
+          accountId
+        )
+      );
+    }
+  }
+
+  return formattedMergedOp;
+};
+
+export const fetchAccountNetworkInfo = async (
+  account: Account
+): Promise<NetworkInfo> => {
+  try {
+    const extendedAccount = await server
+      .accounts()
+      .accountId(account.freshAddress)
+      .call();
+
+    const numberOfEntries = extendedAccount.subentry_count;
+
+    const ledger = await server
+      .ledgers()
+      .ledger(extendedAccount.last_modified_ledger)
+      .call();
+
+    const baseReserve = BigNumber(
+      (ledger.base_reserve_in_stroops * (2 + numberOfEntries)).toString()
+    );
+
+    const fees = BigNumber(ledger.base_fee_in_stroops.toString());
+
+    return {
+      family: "stellar",
+      fees,
+      baseReserve,
+    };
+  } catch (error) {
+    return {
+      family: "stellar",
+      fees: BigNumber(0),
+      baseReserve: BigNumber(0),
+    };
+  }
+};
+
+export const fetchSequence = async (a: Account) => {
+  const extendedAccount = await server.loadAccount(a.freshAddress);
+  return BigNumber(extendedAccount.sequence);
+};
+
+export const fetchSigners = async (a: Account) => {
+  try {
+    const extendedAccount = await server
+      .accounts()
+      .accountId(a.freshAddress)
+      .call();
+    return extendedAccount.signers;
+  } catch (error) {
+    return [];
+  }
+};
+
+export const broadcastTransaction = async (
+  signedTransaction: string
+): Promise<string> => {
+  const transaction = new StellarSdk.Transaction(
+    signedTransaction,
+    StellarSdk.Networks.PUBLIC
+  );
+
+  const res = await server.submitTransaction(transaction, {
+    skipMemoRequiredCheck: true,
+  });
+  return res.hash;
+};
+
+export const buildPaymentOperation = (
+  destination: string,
+  amount: BigNumber
+) => {
+  const formattedAmount = formatCurrencyUnit(currency.units[0], amount, {
```

Comment:
I think it's a bit dangerous to use `formatCurrencyUnit` which is initially meant to user's display into actual logic.

Typically if we forget about disableRounding.. also the formatter may (or may not) have different behavior in future, depending on local etc..

To me we need a version of this function for logic purpose that is stable.

essentially it's `amount.div(BigNumber(10).pow(currency.units[0].magnitude)).toString(10)`

Can you do a local function for now?